core/esmf-test-aspect-models/src/main/resources/valid/samm_1_0_0/org.eclipse.esmf.test/1.0.0/AspectWithConstrainedSet.ttl

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+#
+# See the AUTHORS file(s) distributed with this work for additional
+# information regarding authorship.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+
+@prefix : <urn:samm:org.eclipse.esmf.test:1.0.0#> .
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:1.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:1.0.0#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+
+:AspectWithConstrainedSet a samm:Aspect ;
+   samm:name "AspectWithConstrainedSet" ;
+   samm:preferredName "Test Aspect"@en ;
+   samm:description "This is a test description"@en ;
+   samm:properties ( :testProperty ) ;
+   samm:operations ( ) .
+
+:testProperty a samm:Property ;
+   samm:name "testProperty" ;
+   samm:preferredName "Test Property"@en ;
+   samm:description "This is a test property."@en ;
+   samm:characteristic :TestTrait .
+
+:TestTrait a samm-c:Trait;
+    samm:name "TestTrait" ;
+    samm-c:baseCharacteristic :TestSet;
+    samm-c:constraint :TestSetConstraint.
+
+:TestSet a samm-c:Set ;
+   samm:name "TestSet" ;
+   samm:preferredName "Test Set"@en ;
+   samm:description "This is a test set."@en ;
+   samm:see <http://example.com/> ;
+   samm:dataType xsd:string .
+
+:TestSetConstraint a samm-c:LengthConstraint;
+    samm:name "TestSetConstraint" ;
+    samm:preferredName "Company ID Set Constraint"@en;
```

Comment:
This is oddly specific for a test model; please remove this description and the samm:see